netket/optimizer/jax_stochastic_reconfiguration.py

```diff
@@ -0,0 +1,370 @@
+from functools import partial
+import numpy as _np
+from scipy.linalg import lstsq as _lstsq
+from scipy.linalg import cho_factor as _cho_factor
+from scipy.linalg import cho_solve as _cho_solve
+from scipy.sparse.linalg import LinearOperator
+from netket.stats import sum_inplace as _sum_inplace
+from scipy.sparse.linalg import cg, gmres, minres
+from mpi4py import MPI
+
+from netket.utils import jax_available
+
+if jax_available:
```

Comment:
I would check if jax is available in the __init__.py of the optimizer module, and then if available import jax_stochastic_reconfiguration, so that in this file you are already working with the assumption that jax is there and don't need to check again